docs/source/theoreticalspectrumgenerator.rst

```diff
@@ -117,7 +123,18 @@ which you again can visualize with:
 
 The first example shows how to put peaks of a certain type, y-ions in this case, into
 a spectrum. The second spectrum is filled with a complete fragment ion spectrum
-of all peaks (a-, b-, y-ions and losses). The losses are based on commonly
+of all peaks (a-, b-, y-ions, precursor peaks, and losses).
+
+From the peptide with 9 amino acids, each ion type should be produced theoretically
```

Comment:
I think "each ion type should be produced theoretically with 8 peaks." might not be clear.

Reply:
How about:
Here, from the peptide with 9 amino acids, fragments theoretically can occur in 8 different positions, resulting in 8  peaks per ion type (a, b, and y-ion in this example code).

I'm not trying to explain too much, as this is a tutorial, not a lecture note... What fragmentation of a peptide means is out-of-scope for the tutorial, and

Reply:
I think that works.
Regarding the lecture part... it is a bit difficult to say where to put such information.
Ideally, we would have it either in https://pyopenms.readthedocs.io/en/latest/introduction.html#background
or in this part (e.g., experienced users can just skip over it but newbies might profit from getting a refresher on ion types etc.).
But yeah that is out of the scope of this sprint.

Reply:
Agreed. It'd be great if we have a background page to link this